samples/msal-angular-v2-samples/angular11-b2c-sample/src/app/profile/profile.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { apiConfig } from '../b2c-config';
+
+type ProfileType = {
+  name?: String
+}
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.component.html',
+  styleUrls: ['./profile.component.css']
+})
+export class ProfileComponent implements OnInit {
+  profile!: ProfileType;
+
+  constructor(
+    private http: HttpClient
+  ) { }
+
+  ngOnInit() {
+    this.getProfile(apiConfig.uri);
+  }
+
+  getProfile(url: string) {
+    this.http.get(url)
+      .subscribe(profile => {
+        console.log(profile)
```

Comment:
Nit: should this be left in?

Reply:
hmm, looks like it's the same with other Angular samples 😕

Reply:
Okay, that's fine. Just checking